### ISSUE TITLE
GLU dependency no longer needed?

### ISSUE THREAD

**Issue by cassidyjames** [flathub/org.godotengine.godot.BaseApp#47]
Reported by @davidmhewitt at https://github.com/elliegames/sanctify-game/pull/12#discussion_r1883755667

Upstream commit: https://github.com/godotengine/godot/commit/587826f879a1cf7a6ac578ae5cf5af160782d74f

**Comment by Calinou** [comment]
Indeed, I don't know what GLU would be needed for in Godot 4 (or even 3.x). Feel free to open a PR for this 🙂 

This also applies to the Godot editor Flatpak, by the way.

**Comment by wjt** [comment]
I think this was resolved by https://github.com/flathub/org.godotengine.godot.BaseApp/pull/48, at least for recent engine branches.
